api/controllers/v1/caver/update.js

```diff
@@ -0,0 +1,137 @@
+const ControllerService = require('../../../services/ControllerService');
+const ErrorService = require('../../../services/ErrorService');
+const MappingV1Service = require('../../../services/MappingV1Service');
+const RightService = require('../../../services/RightService');
+const CaverService = require('../../../services/CaverService');
+// eslint-disable-next-line consistent-return
+module.exports = async (req, res) => {
+  // list of propreties to update
+  const propretiesUpdatable = [
+    'name',
+    'nickname',
+    'mail',
+    'password',
+    'surname',
+    'organizations',
+  ];
+
+  // Check right (Admin can't edit mail and password, Caver can only edit itself)
```

Comment:
`(Admin can't edit mail and password, Caver can only edit itself)` à retirer. 

Le principe d'utiliser les droits dans la BD c'est qu'à tout moment on peut en retirer et en enlever. Dans le code, on ne sait pas qui a droit ou pas. On sait juste si l'utilisateur peut ou pas, indépendamment de son rôle.